packages/UI/src/components/Modal/Modal.stories.tsx

```diff
@@ -0,0 +1,527 @@
+    );
+  },
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/EwzuhhvTulvFRMvhTD5VAh/DL-Universal-Design-System?node-id=8769%3A6382',
+    },
+  },
+};
+
+export const EmptyModal: Story = {
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/EwzuhhvTulvFRMvhTD5VAh/DL-Universal-Design-System?node-id=8769%3A6381',
+    },
+  },
+  render: () => {
+    // Empty modal jsx
+    const EmptyTriggerBtn = (
+      <Button buttonType="primary" buttonSize="standard">
+        Empty
+      </Button>
+    );
+    const EmptyHeader = (
+      <h3 className="h-8 w-full bg-grayscale-100 uppercase"> </h3>
+    );
+    const EmptyContent = <div className="h-36 w-full bg-grayscale-100"></div>;
+    const EmptyFooter = <div className="h-10 w-full bg-grayscale-100"></div>;
+
+    // Return all modals
+    return (
+      <div className="flex gap-4">
+        <Modal
+          trigger={EmptyTriggerBtn}
+          header={EmptyHeader}
+          content={EmptyContent}
+          footer={EmptyFooter}
+        />
+      </div>
+    );
+  },
+};
+
+export const Subscribe: Story = {
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/EwzuhhvTulvFRMvhTD5VAh/DL-Universal-Design-System?node-id=8769%3A6380',
+    },
+  },
+  render: () => {
+    const SubscribeTriggerBtn = (
+      <Button buttonType="primary" buttonSize="standard">
+        Subscribe!
+      </Button>
+    );
+    const SubscribeHeader = (
+      <h3 className="w-full font-nunito-sans text-lg uppercase tracking-wider">
+        {' '}
+        Subscribe to our newsletter{' '}
+      </h3>
+    );
+    const SubscribeContent = (
+      <div className="">
+        <p className="w-96 pl-8 font-nunito-sans">
+          {' '}
+          <Description>
+            Subscribe to our newsletter for the latest trends and exclusive
+            content delivered straight to your inbox!
+          </Description>
+        </p>
+        {/* Email address label with an email address input element */}
+        <div className="flex flex-col items-center justify-center">
+          <label className="font-nunito-sans text-sm font-bold uppercase tracking-wide">
+            {' '}
+            Email address{' '}
+          </label>
+          <input
+            className="w-96 rounded-lg border-2 border-solid border-black p-2"
+            type="email"
+            placeholder="Enter your email address"
+          />
+        </div>
+      </div>
+    );
+    const SubscribeFooter = (
+      <div className="flex items-center justify-end gap-6">
+        <Close>
+          <Button buttonType="primary" buttonSize="standard">
+            {' '}
+            dismiss{' '}
+          </Button>
+        </Close>
+        <Close>
+          <Button buttonType="primary" buttonSize="standard">
+            {' '}
+            Subscribe{' '}
+          </Button>
+        </Close>
+      </div>
+    );
```

Comment:
why these jsx elements are inside variables? it doesn't look readable in the code on storybook docs nor serves any purpose in this file.

Reply:
Because it's more readable in storybook than having them directly in the props.